libraries/AP_GPS/AP_GPS_GSOF.cpp

```diff
@@ -371,4 +432,17 @@ AP_GPS_GSOF::validate_com_port(const uint8_t com_port) const {
     }
 }
 
+bool 
+AP_GPS_GSOF::validate_raw_data(const uint8_t raw_data) const {
+    switch(raw_data) {
+        case 0:
```

Comment:
What are these values?

Reply:
> Values: 0:Ignore,1:Always log,2:Stop logging when disarmed (SBF only),5:Only log every five samples (uBlox only) 
They were mentioned in AP_GPS.cpp under parameter descriptions. Anything not these, I labeled them as invalid.